lib/gems/pending/util/extensions/miq-benchmark.rb

```diff
@@ -29,6 +29,10 @@ def self.realtime_block(key, &block)
     begin
       ret = realtime_store(hash, key, &block)
       return ret, hash
+    rescue
+      # Don't let timings be lost on exception when there is nobody else to pick them up.
+      $log.info("Exception in realtime_block #{key.inspect} - Timings: #{hash.inspect}") if outermost
```

Comment:
There are no other log lines in this file.  It probably shouldn't expect `$log` to be defined and instead be initialized with a logger... But, for a surgical change, maybe add `if $log && outermost`

Reply:
It makes perfect sense for a library to write to stderr on errors if they don't provide a facility for specifying a logger.